src/main/cpp/subsystems/DriveSubsystem.cpp

```diff
@@ -65,131 +66,63 @@ DriveSubsystem::DriveSubsystem(Vision *vision)
 }
 
 void DriveSubsystem::SimulationPeriodic() {
-  frc::ChassisSpeeds chassisSpeeds = kDriveKinematics.ToChassisSpeeds(
+  logDrivebase();
+  frc::ChassisSpeeds chassisSpeeds = m_driveKinematics.ToChassisSpeeds(
       m_frontLeft.GetState(), m_frontRight.GetState(), m_rearLeft.GetState(),
       m_rearRight.GetState());
-  m_gyroSimAngle.Set(m_gyro.GetYaw() +
+  m_gyroSimAngle.Set(m_gyro.GetAngle() +
                      (chassisSpeeds.omega.convert<units::deg_per_s>().value() *
                       DriveConstants::kLoopTime.value()));
-  poseEstimator.Update(-m_gyro.GetRotation2d(),
-                    {m_frontLeft.GetPosition(), m_rearLeft.GetPosition(),
-                     m_frontRight.GetPosition(), m_rearRight.GetPosition()});
+  poseEstimator.Update(-m_gyro.GetRotation2d(), GetModulePositions());
```

Comment:
should swap this for a simulated pigeon gyro

Reply:
Make a separate issue?